support-frontend/assets/helpers/contributions.js

```diff
@@ -270,6 +270,8 @@ function toContributionType(s: ?string): ?ContributionType {
         return 'MONTHLY';
       case 'ONE_OFF':
```

Comment:
Do we need a TODO to remember to remove ONE_OFF in the future?

Reply:
I think we can leave it as it might be variably referred to as either